AntiCheat/Lang/en_US.resx

```diff
@@ -238,118 +238,118 @@
     <value>Detected player {player} attempting to call the boss attacking code functions!</value>
   </data>
   <data name="msg_ChatReal" xml:space="preserve">
-    <value>Detected player {player} tried to spoof {player2}'s name!</value>
+    <value>{player} tried to spoof {player2}'s name in chat.</value>
   </data>
   <data name="msg_DespawnItem" xml:space="preserve">
-    <value>Detected player {player} destroying items!</value>
+    <value>{player} was detected destroying items.</value>
   </data>
   <data name="msg_Enemy_ChangeOwnershipOfEnemy" xml:space="preserve">
-    <value>Detected player {player} attempting to change the monster behavior!</value>
+    <value>{player} was detected trying to control a monster.</value>
   </data>
   <data name="msg_Enemy_SwitchToBehaviour" xml:space="preserve">
-    <value>Detected player {player} attempting to change the monster mechanism!</value>
+    <value>{player} was detected trying to alter a monster's behavior.</value>

```

Comment:
im not sure this meaning, so for example the jester has 3 stages, if cheater try to change that will receive this message
im not sure how to say that correctly so i use mechanism word instead